cdap-ui/app/features/workflows/templates/tabs/schedules.html

```diff
@@ -0,0 +1,109 @@
+<!-- HEADING -->
+<div class="schedule-heading row">
+
+  <div class="col-xs-offset-1 col-xs-3">
+    <h4>Schedule Name</h4>
+  </div>
+  <div class="col-xs-4">
+    <h4>Description</h4>
+  </div>
+  <div class="col-xs-2">
+    <h4>Last Run</h4>
+  </div>
+  <div class="col-xs-2">
+    <h4>Status</h4>
+  </div>
+</div>
+
+<div class="schedule-content">
+  <div class="schedule-rows" ng-repeat="schedule in schedules">
+    <div class="row schedule-row"
+      ng-click="toggleLine(schedule)"
```

Comment:
This can be inlined 

```
<div class="..."
        ng-click="schedule.isOpen = !schedule.isOpen"
> ... </div>
```

Feel a function call here is an over kill.